eclass/haskell-cabal.eclass

```diff
@@ -220,43 +242,17 @@ cabal-mksetup() {
 		> "${setup_src}" || die "failed to create default Setup.hs"
 }
 
-cabal-hscolour() {
-	set -- hscolour "$@"
-	echo ./setup "$@"
-	./setup "$@" || die "setup hscolour failed"
-}
-
-cabal-haddock() {
-	set -- haddock "$@"
-	echo ./setup "$@"
-	./setup "$@" || die "setup haddock failed"
-}
-
-cabal-hoogle() {
-	ewarn "hoogle USE flag requires doc USE flag, building without hoogle"
+run_verbose() {
```

Comment:
`run_verbose ()` has a chance of colliding with other eclasses. Please prepend it with an eclass have, like `haskell-cabal_run_verbose ()`

Reply:
Done